gulpfile.js

```diff
@@ -159,6 +161,17 @@ gulp.task('less', function () {
     .pipe(gulp.dest('./modules/'));
 });
 
+// Imagemin task
+gulp.task('imagemin', function () {
+  return gulp.src(defaultAssets.client.img)
```

Comment:
Where are these assets coming from? We don't currently have a definition for *client.img*.